go.mod

```diff
@@ -0,0 +1,31 @@
+module github.com/percona/proxysql_exporter
```

Comment:
Sorry, we can't migrate to Go modules at this point. Please use dep.

Reply:
😭

Reply:
I know, I know… 😄